### PR TITLE
add clean stateful checkpoint handler into tcontroller

### PR DESCRIPTION
**Pull request by nlu90**
This PR added the clean stateful checkpoint handler into tcontroller. Stub method in tmaster will be updated in later pr.

### CODE REVIEW

File: heron/tmaster/src/cpp/manager/tcontroller.cpp [hunk 1]
```diff
@@ -39,6 +39,12 @@ TController::TController(EventLoop* eventLoop, const NetworkOptions& options, TM
   };
 
   http_server_->InstallCallBack("/deactivate", std::move(cbDeActivate));
+
+  auto cbCleanState = [this](IncomingHTTPRequest* request) {
+    this->HandleCleanStatefulCheckpointRequest(request);
+  };
+
+  http_server_->InstallCallBack("/cleanstatefulcheckpoint", std::move(cbCleanState));
```
Comment (srkukarni) [inline heron/tmaster/src/cpp/manager/tcontroller.cpp]: should this be called cleanallstatefulcheckpoints?

Comment (billonahill) [inline heron/tmaster/src/cpp/manager/tcontroller.cpp]: are we able to inject underscores for readability for long, multi-word paths?

Comment (nlu90) [inline heron/tmaster/src/cpp/manager/tcontroller.cpp]: @srkukarni @billonahill  updated the handler name as well as adding underscores

**Comment by srkukarni** [comment]
Looks good to me

**Pull request merged**